ngraph/core/src/op/loop.cpp

```diff
@@ -399,3 +330,28 @@ bool op::v5::Loop::evaluate(const HostTensorVector& outputs, const HostTensorVec
         m_body, m_output_descriptions, m_input_descriptions, m_special_body_ports, outputs, inputs);
     return true;
 }
+
+void op::v5::Loop::clone(op::v5::Loop& dst, const OutputVector& new_args) const
+{
+    dst.set_arguments(new_args);
+    dst.set_output_size(m_output_descriptions.size());
+
+    dst.m_num_iterations = m_num_iterations;
+    dst.m_special_body_ports = m_special_body_ports;
+
+    dst.m_body = clone_function(*get_function());
+
+    for (auto& input_description : m_input_descriptions)
+    {
+        dst.m_input_descriptions.push_back(input_description->copy());
+    }
+    for (auto& output_description : m_output_descriptions)
+    {
+        dst.m_output_descriptions.push_back(output_description->copy());
+    }
+}
+
+op::v5::Loop::Loop(const op::v5::Loop& other)
+{
+    other.clone(*this, other.input_values());
```

Comment:
it looks like this constructor doesn't work as a copy constructor, we don't use information from  `op::v5::Loop& other` except about the inputs

the call `other.clone(*this, other.input_values());` actually copy information from existed loop to this loop:) 
e.g.
` dst.m_num_iterations = m_num_iterations;`
works for this call as
`this.m_num_iterations = this.num_iterations`
so it do nothing.